### ISSUE TITLE
Invalid ident error in switch statement

### ISSUE THREAD

**Issue by tscpp** [eslint/eslint#13345]
**Tell us about your environment**

* **ESLint Version:** v6.8.0
* **Node Version:** v14.3.0
* **npm Version:** 6.14.4

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
	"env": {
		"es6": true,
		"node": true
	},
	"extends": [
		"eslint:recommended",
		"plugin:@typescript-eslint/eslint-recommended"
	],
	"globals": {
		"Atomics": "readonly",
		"SharedArrayBuffer": "readonly"
	},
	"parser": "@typescript-eslint/parser",
	"parserOptions": {
		"ecmaVersion": 2018,
		"sourceType": "module"
	},
	"plugins": [
		"@typescript-eslint"
	],
	"rules": {
		"indent": [
			"error",
			"tab"
		],
		"quotes": [
			"error",
			"single"
		],
		"semi": [
			"error",
			"always"
		],
		"no-unused-vars": "off",
		"@typescript-eslint/no-unused-vars": "warn"
	}
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
...
	const convertCategory = (category: IssueCategory): DiagnosticSeverity => {
		switch (category) {
			case IssueCategory.Error: return DiagnosticSeverity.Error;
			case IssueCategory.Warn: return DiagnosticSeverity.Warning;
			case IssueCategory.Info: return DiagnosticSeverity.Information;
			case IssueCategory.Hint: return DiagnosticSeverity.Hint;
		}
	};
...
```

```bash
 npx eslint
```

**What did you expect to happen?** The switch expression and the case expression and at the same tab level. 

**What actually happened? Please include the actual, raw output from ESLint.** Eslint threw the error `Expected indentation of 2 tabs but found 3`.

**Are you willing to submit a pull request to fix this bug?** No

**Comment by kaicataldo** [comment]
Using the [`Switchcase`](https://eslint.org/docs/rules/indent#switchcase) option should solve your problem.
